package-lock.json

```diff
@@ -109,7 +109,8 @@
       "version": "8.1.0",
       "resolved": "https://registry.npmjs.org/@commitlint/execute-rule/-/execute-rule-8.1.0.tgz",
       "integrity": "sha512-+vpH3RFuO6ypuCqhP2rSqTjFTQ7ClzXtUvXphpROv9v9+7zH4L+Ex+wZLVkL8Xj2cxefSLn/5Kcqa9XyJTn3kg==",
-      "dev": true
+      "dev": true,
+      "optional": true
```

Comment:
Where are the changes in the root-level `package.json` file coming from? Can you revert them please?

Reply:
@bajtos you mean the `package-lock.json`? I think it's just updated by `npm i`
If you mean `package.json`, other than the folder name change in [commit](https://github.com/strongloop/loopback-next/pull/3761/commits/55e80176a0367e9258f86a30c5ff32c65f4b170a), I didn't change anything, they are just copied from the `labs/authentication-passport` folder.

Reply:
Sorry for the confusion, I mean `package-lock.json` file.

I prefer to let RenovateBot update the lock files in standalone pull requests.

Reply:
@bajtos no worries :) There is a breaking change in the 3rd commit(which updates the lock file), the dependency `@loopback/authentication`'s version is updated from 2.x to 3.x, so I feel the change should be included in the commit. Thought?

Reply:
> There is a breaking change in the 3rd commit(which updates the lock file), the dependency `@loopback/authentication`'s version is updated from 2.x to 3.x, so I feel the change should be included in the commit. Thought?

Sure. If you are updating dependencies in `package.json`, then `package-lock.json` needs to be updated too. (Ideally just with the changes needed for updating that single dependency.)

Here I am concerned about updating `package-lock.json` in monorepo root, this files does not seem to list `@loopback/authentication` at all. (It would be very suspicious if it did!)

Reply:
Oh I see. will revert it.

Reply:
@bajtos Good catch I didn't realize the root level one was updated, changes are reverted 👍